index.js

```diff
@@ -16,16 +16,40 @@ function inject (dispatchFunc, options, callback) {
   }
 }
 
+// TODO(mcollina): use standard wrapping instead
+function supportStream1 (req, next) {
+  const payload = req._lightMyRequest.payload
+  if (!payload || payload._readableState || typeof payload.resume !== 'function') { // does not quack like a stream
+    return next()
+  }
+
+  const chunks = []
+
+  payload.on('data', (chunk) => chunks.push(Buffer.from(chunk)))
+
+  payload.on('end', () => {
+    const payload = Buffer.concat(chunks)
+    req.headers['content-length'] = req.headers['content-length'] || ('' + payload.length)
```

Comment:
Make sure you delete the eventual `transfer-encoding` header here since you're setting the content length.